php-classes/Emergence/Connectors/SyncResult.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Emergence\Connectors;
+
+class SyncResult
+{
+    const STATUS_CREATED = 'created';
+    const STATUS_UPDATED = 'updated';
+    const STATUS_SKIPPED = 'skipped';
+    const STATUS_VERIFIED = 'verified';
+    const STATUS_DELETED = 'deleted';
+
+    protected $status;
+    protected $message = '';
+    protected $context = [];
+
+    public function __construct($status, array $options = [])
```

Comment:
@nbey we're going to be requiring message, so make the function signature `$status, $message, array $context = []`